pyproject.toml

```diff
@@ -68,6 +68,7 @@ ignore = [
   "binder/*",
   ".env_sample",
   ".devcontainer/*",
+  "napari/resources/icons/_themes/*/*.svg"
```

Comment:
I also need to fix check-manifest configuration but do not have any idea where I should add a test for it.

Reply:
Hi @Czaki there is a check on every PR 

https://github.com/napari/napari/blob/main/.github/workflows/test_pull_requests.yml#L21

What do you mean 🤔 ?

Reply:
run `make dist` locally, and it ends with:

```
lists of files in version control and sdist do not match!
missing from VCS:
  napari/resources/icons/_themes/dark/2D.svg
  napari/resources/icons/_themes/dark/2D_50.svg
  napari/resources/icons/_themes/dark/3D.svg
  napari/resources/icons/_themes/dark/3D_50.svg
  napari/resources/icons/_themes/dark/add.svg
  napari/resources/icons/_themes/dark/add_50.svg
  napari/resources/icons/_themes/dark/check.svg
  napari/resources/icons/_themes/dark/check_50.svg
  napari/resources/icons/_themes/dark/chevron_down.svg
  napari/resources/icons/_themes/dark/chevron_down_50.svg
  napari/resources/icons/_themes/dark/chevron_left.svg
...
```

So there is a bug in the configuration of the check manifest as importing napari creates multiple files.

Reply:
Could you amend the check manifest test so that we test in the conditions you describe?

Reply:
@goanpeca here you have https://github.com/napari/napari/actions/runs/3092349328/jobs/5003513211

Reply:
awesome!